tools/releaseTools.psm1

```diff
@@ -1,3 +1,5 @@
+#requires -Version 6.0
```

Comment:
Our guidance says the copyright has to be first... This is obviously a reasonable exception, but should we update the guidance?